docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -90,6 +94,7 @@ In this case, the token will move through the gateway and (according to the cond
 
 ## Additional resources and next steps
 
+- [Video-based Human Task Orchestration Course](https://bit.ly/3PJJocB)
```

Comment:
Same comment here 👍

Reply:
Yes, the Academy video is accessible to everyone. 👍